### PR TITLE
Addition of annealus lepton isolations

### PR DESCRIPTION
**Pull request by mmarionncern**
Compute the annealus isolations for each lepton.

### CODE REVIEW

File: PhysicsTools/Heppy/python/analyzers/objects/LeptonAnalyzer.py [hunk 1]
```diff
@@ -200,6 +200,7 @@ def makeLeptons(self, event):
         if self.doIsoAnnulus:
             for lep in event.inclusiveLeptons:
                 self.attachIsoAnnulus04(lep)
+                self.attachIsoAnnealusMiniIso(lep)
```
Comment (gpetruc) [inline PhysicsTools/Heppy/python/analyzers/objects/LeptonAnalyzer.py:203]: why one is Annulus and the other Annealus?
could we use just one spelling, whatever is more correct

**Review by gpetruc — APPROVED** [review]
looks ok to me for the rest

**Comment by gpetruc** [comment]
@mmarionncern does this need to be forward-ported to 94X?

File: PhysicsTools/Heppy/python/analyzers/objects/LeptonAnalyzer.py (hunk 1 above)
Comment (mmarionncern) [reply PhysicsTools/Heppy/python/analyzers/objects/LeptonAnalyzer.py:203]: Yes, going to correct that asap in the different releases

**Comment by mmarionncern** [comment]
@gpetruc yes, it would be good to have it as well in 94X

**Pull request merged**